tools/podio-dump

```diff
@@ -38,7 +38,7 @@ def print_frame(frame, cat_name, ientry, detailed):
   print('Collections:')
 
   if not detailed:
-    print(f'{"Name":<30} {"Type":<40} {"Size":<10}')
+    print(f'{"Name":<38} {"ID":<4} {"Type":<32} {"Size":<10}')
```

Comment:
32 chars for the type might still not be enough for e.g. `edm4hep::MCRecoParticleAssociation`. Ideally this would be dynamically sized according to the longest type. However, we already have #352 for that, so we can keep this as is for now.